src/components/PrimaryNav/PrimaryNav.tsx

```diff
@@ -53,25 +64,25 @@ const ControllersLink = () => {
   }, [controllers]);
 
   return (
-    <NavLink
-      className={({ isActive }) =>
-        classNames("p-list__link", {
-          "is-selected": isActive,
-        })
-      }
-      to={urls.controllers}
-    >
+    <PrimaryNavLink to={urls.controllers}>
       <i className={`p-icon--controllers is-light`}></i>
       <span className="hide-collapsed">Controllers</span>
       {controllersUpdateCount > 0 && (
         <span className="entity-count is-caution">
           {controllersUpdateCount}
         </span>
       )}
-    </NavLink>
+    </PrimaryNavLink>
   );
 };
 
+const LogsLink = () => (
+  <PrimaryNavLink to={urls.logs}>
+    <i className={`p-icon--topic is-light`}></i>
```

Comment:
This could use the [<Icon />](https://canonical.github.io/react-components/?path=/docs/icon--base) component (as could all the icons in this component):

```jsx
<Icon name="topic" light />
```

Reply:
Nice catch! I've updated it in the newly created `PrimaryNavLink` component.